core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -1143,7 +1143,8 @@ private[spark] class BlockManager(
         val buf = blockTransferService.fetchBlockSync(loc.host, loc.port, loc.executorId,
           blockId.toString, tempFileManager)
         if (blockSize > 0 && buf.size() == 0) {
-          throw new IllegalStateException("Empty buffer received for non empty block")
+          throw new IllegalStateException(s"Empty buffer received for non empty block " +
```

Comment:
Small nits: no need for interpolation on first half of string; "fetch" -> "fetching"

Reply:
> Small nits: no need for interpolation on first half of string; "fetch" -> "fetching"

Done